### ISSUE TITLE
Reference image feature

### ISSUE THREAD

**Issue by SWAPv1** [zzzyuqing/DreamMat#7]
Hello I am blown away by how production worthy your tool can be!
Only thing I'd like is a image to texture feature instead of text to texture. 
Is this already possible if not do you have plans for this?

**Comment by zzzyuqing** [comment]
Yes, you can achieve image-to-material by using methods such as the [IP-Adapter](https://github.com/tencent-ailab/IP-Adapter), which is quite easy to implement. However, further testing is required to ensure stability.

**Comment by zhoushiwei** [comment]
> Yes, you can achieve image-to-material by using methods such as the [IP-Adapter](https://github.com/tencent-ailab/IP-Adapter), which is quite easy to implement. However, further testing is required to ensure stability.

Hi , how to use IP-Adapter to merge your project ,can u give some ideas

**Comment by XZYW7** [comment]
basic sd model is loaded in the method `configure(self)` in [dreammat_guidance.py](https://github.com/zzzyuqing/DreamMat/blob/main/threestudio_dreammat/threestudio/models/guidance/dreammat_guidance.py)，you can load ip-adapter or other diffusers supported adapters like lora here
